components/status.tsx

```diff
@@ -8,9 +10,35 @@ export default async function Status({
   description: string;
 }) {
   return (
-    <article className="m-auto max-w-fit text-center">
-      <h2>{title}</h2>
-      <p>{description}</p>
+    <article className="flex h-screen items-center justify-center text-error">
+      <div className="flex items-center space-x-12">
+        <div className="text-center">
+          <h1
+            className="font-fingerpaint text-[190px] leading-none"
+            style={{
+              fontWeight: 400,
+              background: 'linear-gradient(to right, #E7695F, #E13F32)',
+              WebkitBackgroundClip: 'text',
+              color: 'transparent',
+            }}
+          >
+            {title}
+          </h1>
+          <p
+            className="font-poppins text-center text-[40px] leading-[48px] text-[#D27D78]"
+            style={{ fontWeight: 500, width: '361px', height: '48px' }}
+          >
+            {description}
+          </p>
+        </div>
+        <Image
+          src="/assets/error-1.png"
```

Comment:
```suggestion
          src="assets/error-1.png"
```
you've added an extra / , it is prefixed automatically